app/view/settings/SeatView.js

```diff
@@ -963,10 +963,22 @@ SDL.SeatView = Em.ContainerView.create({
             content: 'ID'
         }),
 
-        seatModuleUUID: SDL.Label.create({
+        seatModuleUUID: Em.Select.create({
             elementId: 'seatModelCurrent',
             classNames: 'seatModelCurrent',
-            contentBinding: 'SDL.RCModulesController.currentSeatModuleID'
+            change: function() {
+              console.log('CHANGE!')
+              var desiredUUID = this.selection;
+              var location_name = '';
+              for(var key in SDL.RCModulesController.seatModels) {
+                if(desiredUUID === SDL.RCModulesController.seatModels[key].UUID) {
+                  location_name = SDL.RCModulesController.seatModels[key].ID;
+                  break;
+                }
+              }
+              SDL.RCModulesController.set('currentSeatModel', SDL.RCModulesController.getCoveringModuleModel('SEAT', location_name));
```

Comment:
@YarikMamykin probably you should wrap these two lines into if statement for case if `location_name` is still empty after the loop

Reply:
@AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/13/commits/0e778e6a9d256cfbf9d86957eea0d3a0286d4448